R/geom-label.R

```diff
@@ -88,7 +88,7 @@ GeomLabel <- ggproto("GeomLabel", Geom,
     data <- coord$transform(data, panel_params)
     data$vjust <- compute_just(data$vjust, data$y, data$x, data$angle)
     data$hjust <- compute_just(data$hjust, data$x, data$y, data$angle)
-    if (!is_margin("margin")) {
```

Comment:
What on earth was this supposed to do?

Reply:
If I'm tracing this correctly, this is a merge scar from 3.5.2.